README.md

```diff
@@ -1,26 +1,92 @@
-![Cartopy logo](http://scitools.org.uk/cartopy/docs/latest/_static/cartopy.png)
+<h1 align="center" style="margin:1em;">
+  <a href="http://scitools.org.uk/cartopy/docs/latest/">
+    <img src="http://scitools.org.uk/cartopy/docs/latest/_static/cartopy.png"
+         alt="Cartopy"></a>
+</h1>
 
-Cartopy
--------
-[![Build Status](https://api.travis-ci.org/repositories/SciTools/cartopy.svg?branch=master)](https://travis-ci.org/SciTools/cartopy/branches)
+<h4 align="center">
+    Cartopy is a Python package designed to make drawing maps for
+    data analysis and visualisation easy.
+</h4>
 
-[![DOI](https://zenodo.org/badge/5282596.svg)](https://zenodo.org/badge/latestdoi/5282596)
+<p align="center">
+<!-- https://shields.io/ is a good source of these -->
+<a href="https://anaconda.org/conda-forge/cartopy">
+<img src="https://img.shields.io/conda/dn/conda-forge/cartopy.svg"
+ alt="conda-forge downloads" /></a>
+<a href="https://github.com/SciTools/cartopy/releases">
+<img src="https://img.shields.io/github/tag/SciTools/cartopy.svg"
+ alt="Latest version" /></a>
+<a href="https://github.com/SciTools/cartopy/commits/master">
+<img src="https://img.shields.io/github/commits-since/SciTools/cartopy/latest.svg"
+ alt="Commits since last release" /></a>
+<a href="https://github.com/SciTools/cartopy/graphs/contributors">
+<img src="https://img.shields.io/github/contributors/SciTools/cartopy.svg"
+ alt="# contributors" /></a>
+<a href="https://travis-ci.org/SciTools/cartopy/branches">
```

Comment:
This badge looks the same as the TravisCI badge, both are labelled build which is confusing (particularly since one is failing and one is passing, resulting in a confusing outlook for the state of the package).

Reply:
OK let's lose the CircleCI one then.